src/test/java/com/github/phillima/test/output/TestJSONOutput.java

```diff
@@ -11,162 +12,177 @@
+        aSniffer.collectSingle();
+        String pathSeparator = FileSystems.getDefault().getSeparator();
 
-		String projectName = FileUtils.getProjectName(Path.of(testFilePath));
-		String dirPathResult = testFilePath + pathSeparator + "asniffer_results";
-		assertTrue(new File(dirPathResult + pathSeparator  + "asniffer-CV.json").exists());
-		assertTrue(new File(dirPathResult + pathSeparator + "asniffer-SV.json").exists());
-		assertTrue(new File(dirPathResult + pathSeparator  + "asniffer-PV.json").exists());
+        String projectName = FileUtils.getProjectName(Path.of(testFilePath));
+        String dirPathResult = testFilePath + pathSeparator + "asniffer_results";
+        assertTrue(new File(dirPathResult + pathSeparator + "asniffer-CV.json").exists());
+        assertTrue(new File(dirPathResult + pathSeparator + "asniffer-SV.json").exists());
+        assertTrue(new File(dirPathResult + pathSeparator + "asniffer-PV.json").exists());
 
-		//delete the reports
-		assertTrue(new File(dirPathResult + pathSeparator  + "asniffer-CV.json").delete());
-		assertTrue(new File(dirPathResult + pathSeparator  + "asniffer-SV.json").delete());
-		assertTrue(new File(dirPathResult + pathSeparator  + "asniffer-PV.json").delete());
-		assertTrue(new File(dirPathResult + pathSeparator ).delete());
+        //delete the reports
+        assertTrue(new File(dirPathResult + pathSeparator + "asniffer-CV.json").delete());
+        assertTrue(new File(dirPathResult + pathSeparator + "asniffer-SV.json").delete());
+        assertTrue(new File(dirPathResult + pathSeparator + "asniffer-PV.json").delete());
+        assertTrue(new File(dirPathResult + pathSeparator).delete());
 
-	}
+    }
 
-	@Test
-	public void testSchemaChildShouldNotHaveChildren() {
-		testFilePath = Paths.get(System.getProperty("user.dir") + "/annotationtest/schemaChildTest").toString();
-		report = AmFactory.createAm(testFilePath, "asniffer").calculate();
+    @Test
+    public void testSchemaChildShouldNotHaveChildren() {
+        var schemaChildTestPath = Paths.get(System.getProperty("user.dir") + "/annotationtest/schemaChildTest").toString();
+        var schemaChildReport = AmFactory.createAm(schemaChildTestPath, "asniffer").calculate();
 
-		List<Children> packagesContentReport = ReportTypeUtils.fetchPackages(report.getPackages(), new FetchSystemViewIMP());
+        List<Children> packagesContentReport = ReportTypeUtils.fetchPackages(schemaChildReport.getPackages(), new FetchSystemViewIMP());
 
-		var childrenOfSchema = packagesContentReport.get(0).getChildrens().stream()
+        var childrenOfSchema = packagesContentReport.get(0).getChildrens().stream()
                 .filter(children -> CodeElementType.SCHEMA.equals(children.getType()))
                 .filter(children -> !children.getChildrens().isEmpty())
-				.collect(Collectors.toList());
+                .collect(Collectors.toList());
+
+        assertEquals(0, childrenOfSchema.size());
+    }
```

Comment:
The "testSchemaChildShouldNotHaveChildren" should be put in another test file. A good place would be the class "TestAnnotationSchema".